mysql/resource_grant.go

```diff
@@ -431,6 +432,10 @@ func parseResourceFromData(d *schema.ResourceData) (MySQLGrant, diag.Diagnostics
 
 	// Step 2: Get generic attributes
 	database := d.Get("database").(string)
+	if database == "" {
+		// treat an empty value as a request for global privileges
```

Comment:
Why do we need this with default "*"?
(I'm not strictly against that, just asking, where it helps)

Reply:
yeah, I was dealing with the unit, but, I think the default "*" helps, I can revert this change